Sent upstream 2009-Oct-18

--- origsrc/libX11-1.3.1/specs/troffrules.in	2009-10-16 16:26:29.000000000 -0500
+++ src/libX11-1.3.1/specs/troffrules.in	2009-10-18 17:28:18.904102300 -0500
@@ -46,7 +46,7 @@ MOSTLYCLEANFILES = index.*
 GROFF_DEFS = -dxV="$(PACKAGE_STRING)"
 
 # -e to run through eqn, -t to run through tbl
-GROFF_FLAGS = -e -t -ms $(GROFF_DEFS) $(top_srcdir)/specs/macros.t
+GROFF_FLAGS = -e -t -ms $(GROFF_DEFS) -I$(srcdir) $(top_srcdir)/specs/macros.t
 
 SUFFIXES = .ms .ps .txt .html .pdf
 
